web/src/engine/FetchProvider.ts

```diff
@@ -2,6 +2,7 @@ import { type PlatformInfo, Runtime, DetectPlatform, CreateUnsupportedPlatformEr
 import * as FetchProviderNodeWebkit from './FetchProviderNodeWebkit';
 import * as FetchProviderBrowser from './FetchProviderBrowser';
 import { InternalError, NotImplementedError } from './Error';
+import type { JSONElement } from '../../../node_modules/websocket-rpc/dist/types';
```

Comment:
Maybe it can be even more restrict and import `JSONObject` instead?

Reply:
Fine